src/persistence/graphParser.ts

```diff
@@ -61,6 +61,16 @@ export class GraphParser implements Parser {
      */
     public parse(content: string, mime: string, andThen: ((result: any) => void) | null) {
         if (this.mimeTypes.contains(mime)) {
+
+            let $rdf = require('rdflib');
+            try {
+                $rdf.parse(content, this.graph.getRdfLibStore(), "https://example.org/resource.ttl", mime);
+            } catch (e) {
+                // rdflib cannot parse graphs with undefined prefixes (as are used in the tests).
+                // without this catch, the tests would fail, but not for the reason that is tested.
+                console.log("could not parse to rdfLibGraph");
```

Comment:
It's not okay to just log a message to the console when all hell breaks loose and a graph cannot be parsed. The user won't know what happened. The UI should explain that a graph cannot be parsed (as well as why) when this happens.

Reply:
Seems like you added this line, @AlexanderDeSeranno. Can you somehow make this error visible to users? Alternatively, if it's okay to hide this error from users, can you give your reasoning why?